app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -409,9 +413,26 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         }
 
         if (!browserOverlay.isVisible && session.isYoutubeTV &&
-                event.keyCode == KeyEvent.KEYCODE_BACK) {
-            val escKeyEvent = KeyEvent(event.action, KeyEvent.KEYCODE_ESCAPE)
-            (activity as MainActivity).dispatchKeyEvent(escKeyEvent)
+                event.keyCode == KeyEvent.KEYCODE_BACK &&
+                event.action == KeyEvent.ACTION_DOWN) {
+            webView?.evalJS("""
+                (function () {
+                    return document.activeElement.parentElement.parentElement.id === 'guide-list';
```

Comment:
nit: comment to explain what is happening here. Probably outside of the JS because it doesn't make sense to inject JS comments.